tcms/rpc/api/testexecution.py

```diff
@@ -131,7 +131,15 @@ def history(execution_id):
         :raises PermissionDenied: if missing *testruns.view_testexecution* permission
     """
     execution = TestExecution.objects.get(pk=execution_id)
```

Comment:
the function history() is not listed in __all__ in testexecution.py

Reply:
will add it. BTW what is the point of the `__all__` property?

Reply:
In the API modules controls the order in which methods are documented. For the rest of the meaning:
https://docs.python.org/3/tutorial/modules.html#importing-from-a-package